XMLCoder.xcodeproj/project.pbxproj

```diff
@@ -804,7 +804,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				SKIP_INSTALL = YES;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "$(inherited)";
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```

Comment:
is there a specific reason for changing this in the Xcode project file if the PR is only concerned with the Linux support? This change breaks our compatibility with Xcode 10.0, which we'd like to maintain as long as possible.

Reply:
I had upgraded to Swift 5 to see if that would fix any of the linux errors. But I don't think it is necessary, I'll revert all changes relative to the Swift version